list/source/GridListImageItem.css

```diff
@@ -10,6 +10,9 @@
 .enyo-gridlist-imageitem img {
 	display: block;
 	width: 100%;
+	height: 100%;
+	background-size: 100% 100%;
```

Comment:
This and the line below don't make sense in the context of an `img` tag. `img' tags don't use the background-size property, since their content is an image. Same for background-repeat. Height 100% also seems strange here, as it would likely cause distortion on images that aren't perfectly square.